tripzone/src/components/box/Box.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styles from './Box.module.scss'
+
+const Box = (props) => {
```

Comment:
could you let me know what is this Box component for ? could you put proper naming to components